fdbclient/FileBackupAgent.actor.cpp

```diff
@@ -3447,14 +3462,20 @@ namespace fileBackup {
 					wait(tr->onError(e));
 				}
 			}
-
-			Optional<RestorableFileSet> restorable = wait(bc->getRestoreSet(restoreVersion));
+			state bool incremental = wait(restore.incrementalBackupOnly().getOrThrow(tr));
```

Comment:
The  restore should consistently use either `getOrThrow` or `get`. Which we choose depends on whether we want to support restoring from a backup originally submitted using a version < 7.